docs/source/development/traffic_ops_api/v12/cachegroup_fallbacks.rst

```diff
@@ -0,0 +1,209 @@
+.. 
+.. 
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+.. 
+..     http://www.apache.org/licenses/LICENSE-2.0
+.. 
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+.. 
+
+.. _to-api-v12-cachegroupfallbacks:
+
+Cache Group Fallback
+====================
+
+.. _to-api-v12-cachegroupfallbacks-route:
+
+/api/1.2/cachegroup_fallbacks
+++++++++++++++++++++++++++++++
+
+**GET /api/1.2/cachegroup_fallbacks**
+
+  Retrieve fallback related configurations for a cache group.
+
+  Authentication Required: Yes
+
+  Role(s) Required: None
+
+  **Request Query Parameters**
+
+  Query parameter is mandatory. Either one of the parameters must be used. Both cant be used simultaneously.
```

Comment:
Can i send through both query params to fetch ONE record? the ability to fetch one record is something all our other endpoints support. I.e.

GET /api/1.2/cachegroup_fallbacks?cacheGroupId=&fallbackId=